doc/api/process.md

```diff
@@ -1814,11 +1814,12 @@ This function is only available on POSIX platforms (i.e. not Windows or
 Android).
 This feature is not available in [`Worker`][] threads.
 
-## process.setUncaughtExceptionCaptureCallback(fn)
+## process.setUncaughtExceptionCaptureCallback(owner, fn)
 <!-- YAML
 added: v9.3.0
 -->
 
+* `owner` {symbol}
```

Comment:
`owner` is a symbol so that each domain-like implementation can use a unique key to register their own capture callbacks.